Apps/Sandcastle/gallery/Particle System Tails.html

```diff
@@ -0,0 +1,260 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport"
+          content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
+    <meta name="description" content="Particle systems for rocket and comet tails.">
+    <meta name="cesium-sandcastle-labels" content="Beginner, Showcases, Tutorials">
+    <title>Cesium Demo</title>
+    <script type="text/javascript" src="../Sandcastle-header.js"></script>
+    <script type="text/javascript" src="../../../ThirdParty/requirejs-2.1.20/require.js"></script>
+    <script type="text/javascript">
+        require.config({
+            baseUrl : '../../../Source',
+            waitSeconds : 60
+        });
+    </script>
+</head>
+<body class="sandcastle-loading" data-sandcastle-bucket="bucket-requirejs.html">
+<style>
+    @import url(../templates/bucket.css);
+
+    #toolbar {
+        background: rgba(42, 42, 42, 0.8);
+        padding: 4px;
+        border-radius: 4px;
+    }
+
+    #toolbar input {
+        vertical-align: middle;
+        padding-top: 2px;
+        padding-bottom: 2px;
+    }
+
+    #toolbar .header {
+        font-weight: bold;
+    }
+</style>
+<div id="cesiumContainer" class="fullSize"></div>
+<div id="loadingOverlay"><h1>Loading...</h1></div>
+<div id="toolbar"></div>
+<script id="cesium_sandcastle_script">
+    function startup(Cesium) {
+        'use strict';
+//Sandcastle_Begin
+// Particle Image Grabber
+var particleCanvas;
+function getImage() {
+    if (!Cesium.defined(particleCanvas)) {
+        particleCanvas = document.createElement('canvas');
+        particleCanvas.width = 20;
+        particleCanvas.height = 20;
+        var context2D = particleCanvas.getContext('2d');
+        context2D.beginPath();
+        context2D.arc(8, 8, 8, 0, Cesium.Math.TWO_PI, true);
+        context2D.closePath();
+        context2D.fillStyle = 'rgb(255, 255, 255)';
+        context2D.fill();
+    }
+    return particleCanvas;
+}
+
+// Cesium Viewer Information
+var viewer = new Cesium.Viewer('cesiumContainer', {
+    shouldAnimate : true
+});
+
+viewer.scene.debugShowFramesPerSecond = true;
+
+// particle system information
+var modelMatrix = Cesium.Transforms.eastNorthUpToFixedFrame(Cesium.Cartesian3.fromDegrees(-75.59777, 40.03883));
+
+var centerLocation = new Cesium.Cartesian3(0, 0, 1);
+var cartographicCenterLocation = Cesium.Cartographic.fromCartesian(centerLocation);
+var cartographicIncrease = 800.0;
+cartographicCenterLocation.height += cartographicIncrease;
+centerLocation = Cesium.Cartographic.toCartesian(cartographicCenterLocation);
```

Comment:
Why do we need these values? I think there's probably an easier way to accomplish what you are trying to get at here.

Reply:
See [this Sandcastle example](https://cesiumjs.org/Cesium/Build/Apps/Sandcastle/index.html#c=nVRra9swFP0rIl/igCc7r3Zr07A0HaywbllbBgPDUGSlFpUlI8l5bPS/78qyEydhXxZC0NU99+jonqusiUZrzjZMoxsk2QbNmeFljn9Ue0GXVuFcSUu4ZLoboj+JRPDhcqVu1RZdoRURhoV+1zDBqOVK3suUU2KVPgVkJFUbA7tWl4dNVYp0JnlOLKtTiXzrXScykWtQWGoB8roYR/vvrChM9ETyQrA7YkmUq5QJE3n5M67r1S9Y4hex7F57pkIZ7vQBXX3TOdEWVkQO8Uqr/I69aMZM8K4/GOL4cjS66H8I0WiE43E8vIwvQjSO4xjHvZowYyTl8uXA90Bshq16hG0iTdAfjhtowS3NABjXsVZCtMKsOPHgs6deuLJHwAb1WaFnCiuChlxpzsCk47s9a5CwUjo3ODsh+15Cr7UEeND0JHQSDk0HOm53QObnA1cxZwaTNA3qKZAkd4aBP7WV+/5e7Zd1pi3wqh3V+cpAyNTM7lNqfkRewTiMSZkv+JaJJ/7bnd4fvG8DyNYBnigRLjkAt2KffTuaqeo8MEvzLdzRXxZTlRelZQ+HXFDfngpFXzEttQboM89Z03gwbHE2VL52hPNSWF6I3e1uobi0QevQsG31YQhh8PA4RH0YPvjF496/cD0nwF3rP+1p6d4nQeKxA1SJ6gk3x7sQz3/OvrYaXrStgJdx1myvz2pCX1n6qZkq33FAdMLOxNidYFNf+ZHnhdLWqQ7goVsGbxxm1UTLEggspsY44knUFE1SvkY8vUk6J/9WSQdRQYyBzKoUlcakM51EgD8qE6p6G9/WTAuyc5CsP/3iNzHGkwjC8yqrlFgS3WL8Cw) if you're trying to orient something compared to the plane entity.